packages/database/src/migrations/1744261686398-addEnabledCheckToAuditTrigger.ts

```diff
@@ -0,0 +1,109 @@
+import { QueryInterface } from 'sequelize';
+import { AUDIT_USERID_KEY, AUDIT_PAUSE_KEY } from '@tamanu/constants/database';
+import { SETTINGS_SCOPES } from '@tamanu/constants/settings';
+
+export async function up(query: QueryInterface): Promise<void> {
+  await query.sequelize.query(`
+    CREATE OR REPLACE FUNCTION logs.is_audit_changes_enabled()
+    RETURNS boolean AS $$
```

Comment:
```suggestion
    STABLE PARALLEL SAFE
    RETURNS boolean AS $$
```

Reply:
That enables some optimisations, see here for definitions https://www.postgresql.org/docs/current/sql-createfunction.html